@types/express/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Express {
```

Comment:
Doesn't express have official types? Why not to use them? https://www.npmjs.com/package/@types/express

Reply:
I'm just extending those types to declare `_body` because `admin-bro-expressjs` checks if `_body` exists to throw error on old body parser

Reply:
Ah, yeah, true :facepalm: